explorer/src/components/WorldMap.tsx

```diff
@@ -18,21 +16,25 @@ type MapProps = {
 
 export const WorldMap: React.FC<MapProps> = ({ title, countryData, userLocation, loading }) => {
 
+  const [colorScale, setColorScale] = React.useState();
   const [hasNoContent, setHasNoContent] = React.useState<boolean>(true);
   const [tooltipContent, setTooltipContent] = React.useState<string | null>(null);
   const { mode } = React.useContext(MainContext);
 
-  const colorScale: any = scaleLinear()
-    .domain([0, 1200])
-    // @ts-ignore
-    .range(mode === 'dark' ? ['#ffedea', '#ff5233'] : ['orange', 'red']);
-
-
   React.useEffect(() => {
     if (userLocation || countryData) {
       setHasNoContent(false);
     }
+    if (countryData?.data) {
+      const heighestNumberOfNodes: number = countryData.data.reduce((a, b) => a.nodes > b.nodes ? a : b).nodes;
```

Comment:
What about using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max on the array of nodes per country?

```suggestion
      const heighestNumberOfNodes = Math.max(countryData.data.map(country => country.nodes));
```

Reply:
If it's an array of values you're passing in you might have to do

`Math.max.apply(null, countryData.data.map(country => country.nodes))`

Reply:
True, I guess it could also be:

```
const heighestNumberOfNodes = Math.max(...countryData.data.map(country => country.nodes));
```

Reply:
Updated. thanks.